SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -182,6 +182,7 @@ void ASpatialDebugger::CreateWorkerRegions()
 	// Create new actors for all new worker regions
 	FActorSpawnParameters SpawnParams;
 	SpawnParams.bNoFail = true;
+	SpawnParams.bHideFromSceneOutliner = true;
```

Comment:
Assuming we keep SpatialDebugger and SpatialDebuggerEditor separate, maybe this should be a setting passed as argument to CreateworkerRegions?

Reply:
It's valid for both cases as we do not want to see the worker regions in editor or runtime in the scene outliner. It does not change the current behaviour.